subprojects/gradle-plugin/src/main/java/org/spongepowered/gradle/vanilla/internal/ProvideMinecraftPlugin.java

```diff
@@ -254,7 +254,12 @@ private TaskProvider<DownloadAssetsTask> createAssetsDownload(final MinecraftExt
 
         minecraft.getRuns().configureEach(run -> {
             run.getParameterTokens().put(ClientRunParameterTokens.ASSETS_ROOT, assetsDir);
-            run.getParameterTokens().put(ClientRunParameterTokens.NATIVES_DIRECTORY, gatherNatives.map(x -> x.getDestinationDir().getAbsolutePath()));
+
+            // Since 1.19-pre1, we no longer need the native folder for the run configuration.
+            // All natives are now on the classpath.
+            if(!minecraft.targetVersion().get().nativesOnClasspath()) {
```

Comment:
Is this actually necessary? Mojang hasn't removed the `-D.java.library.path` parameter from their launcher json's JVM args in 1.19-pre1

Reply:
Actually, it wouldn't be necessary, but I find it a bit annoying to have to include additional checks for certain libraries, since not every supplied Native Library has `natives-{os}-[{arch}]` as a classifier. 

e.g
`io.netty:netty-transport-native-epoll:4.1.76.Final:linux-x86_64`
`io.netty:netty-transport-native-epoll:4.1.76.Final:linux-aarch_64`

Reply:
I'm afraid I don't see how that is related to this check? The natives on the classpath and the natives from the copy natives task will do their own thing either way. Really, mojang should add a new `arch` rule for the launcher meta to have architecture-specific dependencies, but we don't have that so we will just have to be disappointed instead.

Reply:
I've gone and changed the native copy task to a sync task and removed the check -- this ensures there isn't any old natives left lying around